addon/lib/Feature.jsm

```diff
@@ -1040,10 +1077,9 @@ class Feature {
         const rootDomainHost = this.getRootDomain(host);
         const rootDomainCurrentHost = this.getRootDomain(currentHost);
 
-        // check if host entity is in the entity list;
-        // TODO bdanforth: improve effeciency of this algo
```

Comment:
WONTFIX. No time. Also, perf tests in tree show [no significant, consistent regressions](https://github.com/biancadanforth/tracking-protection-shield-study/issues/26#issuecomment-363272385).